.ci/build-n-run.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
You should do `chmod +x .ci/build-n-run.sh`

Reply:
I add it in the second line.